src/maintainer/knowledge_base.rst

```diff
@@ -232,10 +230,13 @@ Core Dependency Tree Packages (CDTs)
 ------------------------------------
 
 Dependencies outside of the ``conda-forge`` channel should be avoided (see :ref:`no_external_deps`).
-However, there are a few exceptions: some dependencies are so close to the system
-that they are not packaged with ``conda-forge``. These dependencies have to be satisfied with
-*Core Dependency Tree* (CDT) packages. A CDT package consists of repackaged CentOS binaries from the
-appropriate version, either 6 or 7 depending on user choice and platform. We manage the build of CDT
+However, there are a few exceptions: 
+
+Some dependencies are so close to the system that they are not packaged with ``conda-forge``. 
+These dependencies have to be satisfied with *Core Dependency Tree* (CDT) packages. 
+
+A CDT package consists of repackaged CentOS binaries from the appropriate version, 
```

Comment:
I think we could remove this definition (CDT) here and link directly to the glosary https://conda-forge.org/docs/misc/00_intro.html#glossary, what do you think ?

Reply:
from line number 238 right...yeah great will do that

Reply:
> I think we could remove this definition (CDT) here and link directly to the glosary https://conda-forge.org/docs/misc/00_intro.html#glossary, what do you think ?

btw shall I do that

Reply:
Please don't as requested by Isuru